Tests/Unit/Controller/Application/AbstractApplicationControllerTest.php

```diff
@@ -49,6 +51,13 @@ protected function setUp()
 
         $this->view = $this->prophesize(ViewInterface::class);
         $this->inject($this->controller, 'view', $this->view->reveal());
+
+        $typoscriptService = $this->prophesize(TyposcriptService::class);
+        //Return orginal settings without modification
+        $typoscriptService->mergeFlexFormAndTypoScriptSettings(Argument::any())->will(function ($args) {
```

Comment:
This can be simplified to `->willReturnArgument(0)`.

Reply:
Done